boards/arm/stm32_min_dev/pinmux.c

```diff
@@ -57,6 +57,9 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PA11, STM32F1_PINMUX_FUNC_PA11_USB_DM},
 	{STM32_PIN_PA12, STM32F1_PINMUX_FUNC_PA12_USB_DP},
 #endif /* CONFIG_USB_DC_STM32 */
+#ifdef CONFIG_ADC_1
```

Comment:
Use the nodelabel form for this as we are migrating towards that:

`#if DT_HAS_NODE(DT_NODELABEL(adc1))`

Reply:
Done. Could you point me to the conversation on this change?